os/drivers/testcase/test_tz.c

```diff
@@ -0,0 +1,86 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include "testcase_proto.h"
+#include <tinyara/arch.h>
+#ifdef CONFIG_AMEBAD_TRUSTZONE
+#include <arch/chip/amebad_nsc.h>
+#endif
+
+#include <errno.h>
```

Comment:
Shall we maintain headers as below ?
#include <tinyara/config.h>
##standard headers <errno.h>
##TizenRT specific public headers like <tinyara/xxx.h>
##chip specific headers
##internal headers with " "

Reply:
done